# Generated for NaoQi method ALVisionVideoInput.setResolution
# Set the size of the output image.

# Name under which the G.V.M. is known from ALVideoDevice.
string id
# { 0 = kQQVGA, 1 = kQVGA, 2 = kVGA }
int32 size
---
# true if success
bool value


